app/views/patients/_pledge.html.erb

```diff
@@ -10,7 +10,7 @@
     <div class='col-sm-12'><span class="pledge_modal_category">Patient name:</span> <%= patient.name %></div>
     <div class='col-sm-12'><span class="pledge_modal_category">Patient ID:</span> <%= patient.identifier %></div>
     <div class='col-sm-12'><span class="pledge_modal_category">Pledge amount:</span> $<%= patient.fund_pledge %> </div>
-    <div class='col-sm-12'><span class="pledge_modal_category">Appointment date:</span> <%= patient.appointment_date %></div>
+    <div class='col-sm-12'><span class="pledge_modal_category">Appointment date:</span> <%= patient.appointment_date_display %></div>
```

Comment:
Same deal, probably should be able to swap this to `appointment_date.display_date`.